modules/tinymce/src/core/test/ts/browser/textpatterns/FindInlinePatternTest.ts

```diff
@@ -17,285 +17,423 @@ interface ExpectedPatternMatch {
-    ]);
-  });
-
-  it('TBA: inline * and ** with content before and after', () => {
-    const editor = hook.editor();
-    setContentAndCursor(editor, 'a ***x*** b', [ 0 ], 9);
-    const matches = getInlinePattern(editor, inlinePatterns());
-    assertPatterns(matches, [
-      { pattern: { start: '**', end: '**', format: [ 'bold' ] }, startRng: { start: [ 0, 3 ], end: [ 0, 5 ] }, endRng: { start: [ 0, 6 ], end: [ 0, 8 ] }},
-      { pattern: { start: '*', end: '*', format: [ 'italic' ] }, startRng: { start: [ 0, 2 ], end: [ 0, 3 ] }, endRng: { start: [ 0, 8 ], end: [ 0, 9 ] }}
-    ]);
-  });
-
-  it('TBA: force only ** pattern and test return on not existing *** pattern', () => {
-    const editor = hook.editor();
-    setContentAndCursor(editor, '***x***', [ 0 ], 7);
-    const matches = getInlinePattern(editor, [{ type: 'inline-format', start: '**', end: '**', format: [ 'bold' ] }]);
-    assertSimpleMatch(matches, '**', '**', [ 'bold' ], { start: [ 0, 1 ], end: [ 0, 3 ] }, { start: [ 0, 5 ], end: [ 0, 7 ] });
-  });
-
-  it('TBA: force only ** pattern with leading content and test return on not existing *** pattern', () => {
-    const editor = hook.editor();
-    setContentAndCursor(editor, 'y ***x***', [ 0 ], 9);
-    const matches = getInlinePattern(editor, [{ type: 'inline-format', start: '**', end: '**', format: [ 'bold' ] }]);
-    assertSimpleMatch(matches, '**', '**', [ 'bold' ], { start: [ 0, 3 ], end: [ 0, 5 ] }, { start: [ 0, 7 ], end: [ 0, 9 ] });
-  });
-
-  it('TBA: force only ** pattern with trailing ** text and test return on not existing *** pattern', () => {
-    const editor = hook.editor();
-    setContentAndCursor(editor, 'y ***x*** **', [ 0 ], 9);
-    const matches = getInlinePattern(editor, [{ type: 'inline-format', start: '**', end: '**', format: [ 'bold' ] }]);
-    assertSimpleMatch(matches, '**', '**', [ 'bold' ], { start: [ 0, 3 ], end: [ 0, 5 ] }, { start: [ 0, 7 ], end: [ 0, 9 ] });
-  });
-
-  it('TBA: Check match when input pattern has an empty start value', () => {
-    const editor = hook.editor();
-    setContentAndCursor(editor, 'brb', [ 0 ], 3);
-    const matches = getInlinePattern(editor, inlinePatterns());
-    assertPatterns(matches, [
-      { pattern: { start: '', end: 'brb', value: 'be right back' }, startRng: { start: [ 0, 0 ], end: [ 0, 3 ] }, endRng: { start: [ 0, 0 ], end: [ 0, 3 ] }}
-    ]);
-  });
-
-  it('TBA: Check match when input pattern has an empty end value', () => {
-    const editor = hook.editor();
-    setContentAndCursor(editor, 'irl', [ 0 ], 3);
-    const matches = getInlinePattern(editor, inlinePatterns());
-    assertPatterns(matches, [
-      { pattern: { start: '', end: 'irl', value: 'in real life' }, startRng: { start: [ 0, 0 ], end: [ 0, 3 ] }, endRng: { start: [ 0, 0 ], end: [ 0, 3 ] }}
-    ]);
-  });
+const getInlinePatternSetFor = (hook: TinyHooks.Hook<Editor>) => Thunk.cached((): InlinePatternSet => {
```

Comment:
Likewise here, functions should not be outside the root `describe` block

- [ ] Fix test code style so that all test functions are within the describe/context block they apply to

Reply:
@tiny-ben-tran looks like this hasn't been fixed yet.